doc/nrf/ug_thingy53.rst

```diff
@@ -295,60 +272,158 @@ You can program the precompiled firmware image using one of the following ways:
-  These samples can share common USB product name, vendor ID, and product ID.
-  If a sample supports additional USB classes or more than one instance of USB CDC ACM, it must use dedicated product name, vendor ID, and product ID.
+* :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT`
+* :kconfig:option:`CONFIG_IMG_MANAGER`
+* :kconfig:option:`CONFIG_MCUBOOT_IMG_MANAGER`
+* :kconfig:option:`CONFIG_IMG_ERASE_PROGRESSIVELY`
 
-* DFU over-the-air through Simple Management Protocol over Bluetooth.
-  The application acts as GATT server and allows the connected Bluetooth Central to perform the firmware update for both application and network core.
-  The Bluetooth configuration is updated to allow quick DFU data transfer.
-  The application supports Simple Management Protocol (SMP) handlers related to:
+In addition, set the :kconfig:option:`CONFIG_UPDATEABLE_IMAGE_NUMBER` option to ``2`` and :kconfig:option:`CONFIG_MCUBOOT_IMAGE_VERSION` to ``"1.0.0+0"``.
 
-  * Image management
-  * Operating System (OS) management -- used to reboot device after firmware upload is complete.
-  * Erasing settings partition -- needed to ensure that a new application would not be booted with incompatible content in the settings partition that was written by the previous application.
+See :file:`thingy53_nrf5430_cpuapp.conf` in a compatible sample for how this is done.
 
-.. tip::
-   To use your application with Thingy:53 preprogrammed bootloader, you only must set proper configuration of Partition Manager and MCUboot.
-   Other mentioned features are nice to have, but they are not mandatory.
+.. _thingy53_app_usb:
 
-   All of the Thingy:53-compatible applications in the |NCS| also support the mentioned additional features.
-   Refer to the source code of the samples and applications for examples of implementation.
+USB
+===
+
+An application compatible with the Nordic Thingy:53 is expected to enable USB CDC ACM as back-end for logging.
+The logs are provided using USB CDC ACM to allow accessing them without additional hardware.
+
+Most of the applications and samples compatible with the Nordic Thingy:53 use only a single instance of USB CDC ACM that works as the logger's back-end.
+No other USB classes are used.
+These samples can share a common USB product name, vendor ID, and product ID.
+If a sample supports additional USB classes or more than one instance of USB CDC ACM, it must use a dedicated product name, vendor ID, and product ID.
+
+To enable the USB device stack and the CDC ACM class:
+
+* Use the options :kconfig:option:`CONFIG_USB_DEVICE_STACK` and :kconfig:option:`CONFIG_USB_CDC_ACM`.
+* Set the default log level to Error by using options :kconfig:option:`CONFIG_USB_CDC_ACM_LOG_LEVEL_ERR` and :kconfig:option:`CONFIG_USB_DEVICE_LOG_LEVEL_ERR`.
+* Enable the UART log back-end by using the :kconfig:option:`CONFIG_LOG_BACKEND_UART` option and disable SEGGER RTT by using both :kconfig:option:`CONFIG_LOG_BACKEND_RTT` and :kconfig:option:`CONFIG_LOG_BACKEND_RTT`.
```

Comment:
Can you somehow get rid of the "...by using..." structure?

Reply:
replaced "by using" with "with".